src/liboslexec/oslexec_pvt.h

```diff
@@ -2502,6 +2503,37 @@ struct NoiseParams {
 };
 
 
+// Mangle the group and layer into a unique function name
+static std::string
```

Comment:
I get "unused function" errors for these three static functions, presumably because they're not called in all translation units that include oslexec_pvt.h.

Reply:
Thanks, should be fixed now.